### ISSUE TITLE
Interactive graph lists data in different order

### ISSUE THREAD

**Issue by MikeMcC399** [corona-warn-app/cwa-website#2291]
## Where to find the issue

https://www.coronawarn.app/en/analysis/
Transmitted and registered test results
- PCR tests
- Rapid antigen tests

## Describe the issue

In the tab
- PCR tests
- Rapid antigen tests

the axis key below the graph are shown in the order from left to right
1. Positive
2. Negative
3. Invalid

The order from top to bottom of the interactive view varies

1. Daily
    1. Invalid
    2. Negative
    3. Positive
2. Weekly
as for daily
3. 7-day moving average
    1. Positive
    2. Negative
    3. Invalid
4. Cumulative
as for daily

![Analysis data order](https://user-images.githubusercontent.com/66998419/148522144-f6948365-9f29-4786-b59c-18822ca9c8ae.jpg)

## Suggested change

Make the order of data display the same from left to right as from top to bottom.

In other words, for Daily, Weekly and Cumulative make the order from top to bottom
    1. Positive
    2. Negative
    3. Invalid

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks for the suggestion. On the other hand, in the stacked bar graphs, the current order represents the order of stacking.

**Comment by MikeMcC399** [comment]
@dsarkar 

Thank you for drawing my attention to the order of stacking. That makes perfect sense to keep them aligned.

In that case I withdraw my suggestion. 🙂
